runtime/src/rpc/dispatcher.rs

```diff
@@ -211,4 +217,20 @@ impl Dispatcher {
             },
         }
     }
+
+    // Handle key manager policy update.
+    pub fn handle_km_policy_update(&self, signed_policy_raw: Vec<u8>) -> () {
+        match &self.km_policy_handler {
+            Some(handler) => handler(signed_policy_raw),
```

Comment:
Could simplify this by just saying `self.km_policy_handler.and_then(|handler| handler(signed_policy_raw));`.